common/memtest.py

```diff
@@ -13,7 +14,8 @@ def initialize(cls, file_path):
         if os.path.exists(MemTracker.log_file_path):
             os.remove(MemTracker.log_file_path)
         with open(MemTracker.log_file_path, "a") as f:
-            f.write("Beggining memory tracking.\n\n")
+            now = dt.datetime.now()
+            f.write("Relative heap set. Ready for memory tracking (TimeStamp: " + str(now) + ")\n\n")
```

Comment:
In the future, let's switch to using Python's logging library. For example, it can look something like: https://github.com/mikeaboody/phishing-research/blob/d0bba873c2208f3d24a8104c1c5ff5201d04faae/broScripts/parse_pcap.py#L44-L52. Let me merge in https://github.com/mikeaboody/phishing-research/pull/37 first though.